packages/scandipwa/src/route/MyAccount/MyAccount.component.js

```diff
@@ -166,6 +167,10 @@ export class MyAccount extends Component {
             return this.renderLoginOverlay();
         }
 
+        if (!tabMap[activeTab]) {
+            return <NoMatch />;
+        }
```

Comment:
The wishlist page availability is configured from the admin panel.
Please fetch this setting from the BE and check it here.